tests/evaluation/multiple_inequalities/multiple_inequalities.dl

```diff
@@ -0,0 +1,13 @@
+// Souffle - A Datalog Compiler
+// Copyright (c) 2020, The Souffle Developers. All rights reserved
+// Licensed under the Universal Permissive License v 1.0 as shown at:
+// - https://opensource.org/licenses/UPL
+// - <souffle root>/licenses/SOUFFLE-UPL.txt
+// Test that multiple signed inequalities are not indexed with the B-Tree index
```

Comment:
Could you briefly expand on this a little? I.e., this comment will read by people who may not know why this is a problem.